offchain/host-runner/src/main.rs

```diff
@@ -32,13 +32,12 @@ fn log_result<T, E: std::error::Error>(name: &str, result: Result<T, E>) {
 
 #[actix_web::main]
 async fn main() {
-    let filter = EnvFilter::builder()
-        .with_default_directive(LevelFilter::INFO.into())
-        .from_env_lossy();
-    tracing_subscriber::fmt().with_env_filter(filter).init();
+    let config: Config = CLIConfig::parse().into();
+
+    logs::initialize_logs(&config.logs_config);
+    info!("Starting Host Runner");
```

Comment:
```suggestion
    info!(?config, "Starting Host Runner");
```
And then we can remove line 40